openevolve/evaluator.py

```
@@ -16,9 +16,11 @@
 from pathlib import Path
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 import traceback
-
+import datetime
 from openevolve.config import EvaluatorConfig
+from openevolve.database import ProgramDatabase
 from openevolve.evaluation_result import EvaluationResult
+from openevolve.database import ProgramDatabase
```

Ruff (0.11.9)

Redefinition of unused `ProgramDatabase` from line 21

Remove definition: `ProgramDatabase`

(F811)